cloudinary-core/src/main/java/com/cloudinary/Api.java

```diff
@@ -55,7 +55,13 @@ public ApiResponse ping(Map options) throws Exception {
 
     public ApiResponse usage(Map options) throws Exception {
         if (options == null) options = ObjectUtils.emptyMap();
-        return callApi(HttpMethod.GET, Arrays.asList("usage"), ObjectUtils.emptyMap(), options);
+        Map params = ObjectUtils.only(options, "date");
+        Object date = params.get("date");
+        if (date instanceof Date) {
```

Comment:
What if it's a String in the right format?

Reply:
It will pass through as is directly to the request (as will any field called "date" that isn't a java `Date` object).

I added a test to be on the safe side.